### PR TITLE
Add content to FAQ page

### PR DESCRIPTION
**Pull request by KrashLeviathan**
## Problem / Feature

Added content to the FAQ page. There's a Back to Top button that sticks to the bottom right on the page as the user scrolls down.
## What To Test

Check out the FAQ page.
## Dependencies

Issue #12 
## Reviewers

@Skyman12 @csteamengine @KrashLeviathan @CivBase @regalcat @MichaelRupertDev

### CODE REVIEW

File: prisum/static/js/smoothscroll.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/**
```
Comment (CivBase) [inline prisum/static/js/smoothscroll.js]: I recommend putting this is `prisum/static/vend/smoothscroll.min.js`.

File: prisum/static/css/global.css [hunk 2]
```diff
@@ -27,3 +27,10 @@ body {
 #footer .btn-social-icon {
     margin-left: 5px;
 }
+
+.back-to-top {
+    position: fixed;
+    bottom: 12px;
+    right: 2vw;
+    display: none;
+}
```
Comment (CivBase) [inline prisum/static/css/global.css]: I'm a stickler for line breaks.

File: prisum/templates/faq.html [hunk 3]
```diff
@@ -4,6 +4,157 @@
...
+    </p>
+
+    <div style="margin-bottom:60px"></div>
+
+    <!--<h2><a href="#" class="back-to-top">Back to Top</a></h2>-->
+    <a class="btn btn-primary back-to-top" href="#" role="button">Back to Top</a>
 </div>
-{% endblock %}
+
+<!--JAVASCRIPT FOR SMOOTH SCROLLING-->
```
Comment (CivBase) [inline prisum/templates/faq.html:157]: Javascript can actually go in a "script" block using jinja.

``` html
{% block content %}
...
{% endblock %}
{% block script %}
<!--JAVASCRIPT FOR SMOOTH SCROLLING-->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
<script type="text/javascript" src="../static/js/smoothscroll.js"></script>
<script type="text/javascript" src="../static/js/backtotop.js"></script>
{% endblock %}
```

**Comment by KrashLeviathan** [comment]
@CivBase ready for re-review.

**Pull request closed without merging**